src/libpriv/rpmostree-core.c

```diff
@@ -1066,20 +1066,21 @@ find_pkg_in_ostree (OstreeRepo     *repo,
                     gboolean       *out_selinux_match,
                     GError        **error)
 {
-  g_assert (repo);
-
+  gboolean in_ostree = FALSE;
+  gboolean selinux_match = FALSE;
   g_autofree char *cached_rev = NULL;
   g_autofree char *cachebranch = rpmostree_get_cache_branch_pkg (pkg);
+
+  /* NB: we're not using a pkgcache yet in the compose path */
+  if (repo == NULL)
+    goto happy_out; /* Note early happy return */
```

Comment:
I've called this `goto done;` in other places, there's a hit or two for it in `git grep` in both ostree/rpm-ostree.